internal/pkg/cli/job_init.go

```diff
@@ -158,10 +160,23 @@ func (o *initJobOpts) Validate() error {
 
 // Ask prompts for fields that are required but not passed in.
 func (o *initJobOpts) Ask() error {
-	if err := o.askJobType(); err != nil {
+	if err := o.askJobName(); err != nil {
 		return err
 	}
-	if err := o.askJobName(); err != nil {
+	localMft, err := o.mftReader.ReadWorkloadManifest(o.name)
+	if err == nil {
+		jobType, err := localMft.WorkloadType()
+		if err != nil {
+			return fmt.Errorf("read type for service %s from local manifest: %w", o.name, err)
+		}
+		o.wkldType = jobType
+		return nil
```

Comment:
Should we log anything in this scenario saying that the file already exists so we are skipping prompts?

Reply:
Agree-- I think it's likely users won't know/remember/realize the manifest is (still) there.